tools/archive_product.sh

```diff
@@ -30,8 +26,8 @@ else
     PRINTER="cat"
 fi
 
-(xcodebuild -workspace 'Gureum.xcworkspace' -scheme 'OSX' -configuration "${CONFIGURATION}" | $PRINTER) && \
-productbuild --product "tools/preinst.plist" --component "${BUILT_PRODUCTS_DIR}/${PRODUCT_NAME}.app" '/Library/Input Methods' --sign "${INSTALLER_KEY}" ~/Downloads/"${PACKAGE_NAME}.pkg"
-#tar -zcf "${PACKAGE_NAME}.app.tar.gz" "${PRODUCT_NAME}.app"
+(xcodebuild -workspace 'Gureum.xcworkspace' -scheme 'OSX' -configuration "${CONFIGURATION}" archive | $PRINTER) && \
+    echo "Archive dir path is required for further packaging (try paste)" && echo "Archive directory path>" && read -r archive_dir
```

Comment:
이 뒤에 `notarize_product.sh`를 실행하도록 되어 있으니 여기에서 오류가 나면 멈추는 게 좋을 것 같습니다.
```suggestion
    echo "Archive dir path is required for further packaging (try paste)" && echo "Archive directory path>" && read -r archive_dir || \
    exit $?
```